ambari-server/src/main/java/org/apache/ambari/server/bootstrap/BSRunner.java

```diff
@@ -179,6 +179,10 @@ private long calculateBSTimeout(int hostCount) {
     return Math.max(HOST_BS_TIMEOUT, HOST_BS_TIMEOUT * hostCount / PARALLEL_BS_COUNT);
   }
 
+  private String isValidateHostOperation() {
+    return validationInstance ? "Validation" : "Bootstrap";
```

Comment:
Name suggests the method returns `boolean`.

Reply:
Also - why not make these enums (or static at the very least)